src/ResponsiveProvider.js

```diff
@@ -171,6 +180,7 @@ ResponsiveProvider.propTypes = {
         ...breakpointsPropTypes,
         _initial: PropTypes.string,
     }),
+    mobileBreakpoint: PropTypes.oneOf(['xs', 'sm', 'md', 'lg', 'xl']),
```

Comment:
This shouldn't be hardcoded, you don't know how the developer wants his breakpoints, he might have an extra one called "xxs" and move all the others a bit upwards

Reply:
What do you mean by that? Because for what  I can see in the proptypes for example initialMediaType:

```js
initialMediaType: PropTypes.oneOf([
        '_initial',
        'xs',
        'sm',
        'md',
        'lg',
        'xl',
    ]),
```

it cant have xxs, why this prop would be different than the others?

Reply:
Lets go forward with it and redo the defaults in a future major